packages/ui/src/components/TextArea/index.tsx

```diff
@@ -0,0 +1,297 @@
+   */
+  helper?: ReactNode
+  rows?: number
+  minLength?: number
+  maxLength?: number
+  tooltip?: string
+  required?: boolean
+  'data-testid'?: string
+  name?: string
+  onFocus?: DOMAttributes<HTMLTextAreaElement>['onFocus']
+  onBlur?: DOMAttributes<HTMLTextAreaElement>['onBlur']
+  clearable?: boolean
+  labelDescription?: ReactNode
+}
+
+/**
+ * This component offers an extended textarea HTML
+ */
+export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
+  (
+    {
+      id,
+      className,
+      tabIndex,
+      value,
+      onChange,
+      placeholder,
+      rows = 3,
+      disabled = false,
+      readOnly = false,
+      success,
+      error,
+      helper,
+      minLength,
+      maxLength,
+      tooltip,
+      label,
+      autoFocus,
+      required = false,
+      'data-testid': dataTestId,
+      name,
+      onFocus,
+      onBlur,
+      clearable = false,
+      labelDescription,
+    },
+    ref,
+  ) => {
+    const localId = useId()
+
+    // Avoid conflicts between properties
+    const computedReadOnly = !disabled && readOnly
+    const computedSuccess = !disabled && !readOnly ? success : undefined
+    const computedError = !disabled && !readOnly && !success ? error : undefined
+    const computedHelper = !success && !error ? helper : undefined
+
+    return (
+      <Stack gap={1} className={className}>
+        <Stack direction="row" gap="1" alignItems="center">
+          <Stack direction="row" gap="0.5" alignItems="start">
+            <Text
+              as="label"
+              variant="bodySmallStrong"
+              sentiment="neutral"
+              htmlFor={id ?? localId}
+            >
+              {label}
+            </Text>
+            {required ? <Icon name="asterisk" color="danger" size={8} /> : null}
+          </Stack>
+          {labelDescription ?? null}
+        </Stack>
+        <Tooltip text={tooltip}>
+          <StyledTextAreaWrapper>
+            <StyledTextArea
+              aria-invalid={!!computedError}
+              id={id ?? localId}
+              tabIndex={tabIndex}
+              autoFocus={autoFocus}
+              disabled={disabled}
+              rows={rows}
+              ref={ref}
+              value={value}
+              onChange={event => {
+                onChange(event.currentTarget.value)
+              }}
+              isSuccess={!!computedSuccess}
+              isError={!!computedError}
+              isClearable={!!clearable}
+              minLength={minLength}
+              maxLength={maxLength}
+              placeholder={placeholder}
+              data-testid={dataTestId}
+              name={name}
+              onFocus={onFocus}
+              onBlur={onBlur}
+              readOnly={computedReadOnly}
+            />
+            <StyledTextAreaAbsoluteStack
+              direction="row"
+              alignItems="center"
+              gap="1"
+            >
+              {clearable ? (
+                <Button
```

Comment:
A bit miss aligned 
<img width="95" alt="Screenshot 2023-12-11 at 10 17 37" src="https://github.com/scaleway/ultraviolet/assets/15812968/3d89b6d0-11cd-4aae-8a45-b9f88d8e833c">

Reply:
Yes, should be 12px and not 16